experiments/AMIP/components/atmosphere/climaatmos_extra_diags.jl

```diff
@@ -0,0 +1,74 @@
+# these extensions add extra diagnostics to the atmos model output
+import ClimaAtmos.Diagnostics as CAD
+
+import ClimaAtmos.Diagnostics: add_diagnostic_variable!
+
+add_diagnostic_variable!(
```

Comment:
how/where are these meant to be used? a little more explanation in a docstring would be nice :)

Reply:
This is exercised in the buildkite job `slabplanet_atmos_diags`, and I've added more info on where in Atmos this is being called.